app-scroll-effects/effects/toggle-class.html

```diff
@@ -0,0 +1,46 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../app-scroll-effects-behavior.html">
+
+<script>
+  /**
+   * Toggles a CSS class when a `scrollTop` threshold is reached. For example:
+   *
+   * ```html
+   * <x-element effects="toggle-class"
+   *   effects-config='{"toggle-class": { "className": "fixed", "minScrollTop": 100 }}'>
+   * </x-element>
+   * ```
+   * The above code will toggle the class `fixed` when the `scrollTop` is at least 100px.
+   */
+  Polymer.AppLayout.registerEffect('toggle-class', {
+    /**
+     * @param {{className: string, minScrollTop: number}} config
+     * @this Polymer.AppLayout.ElementWithBackground
+     */
+    setUp: function setUp(config) {
+      this._fxToggleClass = config;
+    },
+
+    /**
+     * @this Polymer.AppLayout.ElementWithBackground
+     */
+    run: function run(p, y) {
+      var config = this._fxToggleClass;
+      this.toggleClass(config.className, y >= config.minScrollTop);
```

Comment:
Should we have a default className (e.g. `scrolled`) so you don't need to specify one?